angular2-jwt.ts

```diff
@@ -304,56 +304,34 @@ export function tokenNotExpired(tokenName = AuthConfigConsts.DEFAULT_TOKEN_NAME,
   return token != null && !jwtHelper.isTokenExpired(token);
 }
 
-export const AUTH_PROVIDERS: Provider[] = [
```

Comment:
There doesn't seem to be something in the documentation. However, it is a well known issue and libraries that switched to support AoT made this change. See this issue: https://github.com/angular/angular/issues/11262